skyportal/handlers/api/shift.py

```diff
@@ -0,0 +1,146 @@
+from sqlalchemy.orm import joinedload
+from marshmallow.exceptions import ValidationError
+from baselayer.app.access import permissions, auth_or_token, AccessError
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    Group,
+    Shift,
+)
+
+
+class ShiftHandler(BaseHandler):
+    @permissions(["Upload data"])
```

Comment:
Should be a permission to "Manage shifts"

Reply:
added